cypress/support/get-iframe-content.ts

```diff
@@ -4,17 +4,22 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
+import { RendererType } from '../../src/components/render-page/window-post-message-communicator/rendering-message'
```

Comment:
Is it good practise to import src-code into tests?

Reply:
Normally: No.
But imho in this case it's okay, because it's a test attribute.